src/ntimporters/asana/importer.py

```diff
@@ -2,18 +2,19 @@
 
 from typing import Optional
 
-import asana
-from asana.error import AsanaError
-from dateutil.parser import isoparse
-
 import openapi_client as nt
+from dateutil.parser import isoparse
 from openapi_client import apis, models
 from openapi_client.exceptions import OpenApiException
 
+import asana
+from asana.error import AsanaError
+
 SPEC = {
     "code": "asana",  # codename / ID of importer
     "name": "Asana",  # name of application
     "url": "https://developers.asana.com/docs/asana",  # link to documentation / specs / API
+    "input_fields": ("nt_auth_token", "auth_token"),
```

Comment:
If we want to list input fields in `SPEC`, then we should add `team_id` here as well. If we treat `team_id` as required by default and skip it in `input_fields` then why can't we do the same with `nt_auth_token` and `auth_token`? Is there a case where we wouldn't require them?

Reply:
you're right, i just added